mmdeploy/apis/openvino/utils.py

```diff
@@ -50,3 +51,22 @@ def get_input_info_from_cfg(deploy_cfg: mmcv.Config) -> Dict[str, List]:
                 input_info = dict(zip(input_names, input_info))
             input_info = update_input_names(input_info, input_names)
     return input_info
+
+
+def get_mo_options_from_cfg(
+        deploy_cfg: Optional[mmcv.Config]) -> ModelOptimizerOptions:
```

Comment:
Should `deploy_cfg` have default `None` value just as described in docstring?
`deploy_cfg: Optional[mmcv.Config]=None`

Reply:
I suppose it would be better to remove `None` from the possible values.
Functions like `get_*_from_cfg` assumes, that a config should be passed to it.